src/essentials/Colors/Colors.ts

```diff
@@ -92,7 +92,7 @@ export const SemanticColors = {
             default: Colors.green[900]
         },
         warning: {
-            default: Colors.blue.primary[900]
+            default: Colors.yellow[900]
```

Comment:
I saw this value was inconsistent with out previous colors and "corrected" it, let me know if the change doesn't make sense 🙏🏽

Reply:
all good, definitely a bug here